DESCRIPTION

```diff
@@ -46,7 +46,25 @@ Config/renv/profiles/dev/dependencies:
     rmarkdown,
     RJDBC
 Config/renv/profiles/reporting/dependencies:
+    DBI,
+    digest,
+    DT,
+    fastmap,
+    ggplot2,
+    htmltools,
+    htmlwidgets,
+    kableExtra,
+    grid,
+    gridExtra,
+    gtools,
     leaflet,
+    magrittr,
```

Comment:
This was a dependency to install DT, but never actually loaded in performance.qmd